packages/foundry/src/create.ts

```diff
@@ -0,0 +1,68 @@
+import { basename } from "node:path";
+
+import { cloneProjectTemplate } from "./cloneProjectTemplate.js";
```

Comment:
imported files should have a `.ts` file extension.

Ideally, tooling would give a warning when a `.js` extension is used instead.